<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Create a Ground Truth ROI that is the average of 3 seperate ROIS:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select the Three ROIS:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_ROI1">
         <property name="text">
          <string>ROI 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="OrsObjectClassComboBox" name="comboBox_ROI1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_ROI2">
         <property name="text">
          <string>ROI 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="OrsObjectClassComboBox" name="comboBox_ROI2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_ROI1">
       <item>
        <widget class="QLabel" name="label_ROI3">
         <property name="text">
          <string>ROI 3: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="OrsObjectClassComboBox" name="comboBox_ROI3"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Name of Ground Truth ROI:                             </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_GroundTruthName">
         <property name="text">
          <string>Ground_Truth_ROI</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Compute">
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrsObjectClassComboBox</class>
   <extends>QComboBox</extends>
   <header>ORSServiceClass.ORSWidget.orsobjectclasscombobox.orsobjectclasscombobox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
